File: Data Structures & Algorithms/sort-colors/submission-0.py
class Solution:
    def sortColors(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        buckets = [0, 0, 0]

        for num in nums:
            buckets[num] += 1
        
        index = 0

        for i in range(len(buckets)):
            count = buckets[i]
            for j in range(count):
                nums[index] = i
                index += 1
                

File: Data Structures & Algorithms/two-integer-sum/submission-0.cs
public class Solution {
    public int[] TwoSum(int[] nums, int target)
    {
        var numDict = new Dictionary<int, int>();
        var index = 0;

        foreach (var num in nums)
        {
            if (numDict.ContainsKey(target - num))
            {
                return new int[] {numDict[target - num], index};
            }
            
            numDict.Add(num, index);
            index++;
        }

        return null;
    }
}
